Update _index.md (#804)
mattermost/mattermost-developer-documentation: site/content/integrate/getting-started/how-should-i-integrate.md

To build a richer bot integration, you can make full use of the [Mattermost REST API](/integrate/rest-api/). Everything that you see a Mattermost client doing, your integration can do too with this API.

### I want to build an interactive app

> That's no problem. Our apps framework supports apps written in any language that supports HTTP, and work on our Mobile and Desktop clients. Apps can be hosted anywhere, on a Digital Ocean droplet, AWS EC2, or a server in your data centre. Take a look at our [Quickstart Guide](https://developers.mattermost.com/integrate/apps/quick-start-go/).

Author: ```suggestion
That's no problem. Our Apps provide a framework with utilities like a KV store, bot account creation, OAuth permission scopes, and rich UX entry points.

Apps can be written in any language that supports HTTP, and work on our Mobile and Desktop clients. Apps can be hosted anywhere, on a Digital Ocean droplet, AWS EC2, or a server in your data centre. Take a look at our [Quickstart Guide](https://developers.mattermost.com/integrate/apps/quick-start-go/).
```

> That's no problem. Our apps framework supports apps written in any language that supports HTTP, and work on our Mobile and Desktop clients. Apps can be hosted anywhere, on a Digital Ocean droplet, AWS EC2, or a server in your data centre. Take a look at our [Quickstart Guide](https://developers.mattermost.com/integrate/apps/quick-start-go/).

Reviewer: Slash command and webhooks also use HTTP as the transport protocol and can to hosted anywhere. It point is that Apps provide a _framework_ with utils like a KV store, bot account creation  and OAuth permission scopes and -most notable- rich UX entry points.

To build a richer bot integration, you can make full use of the [Mattermost REST API](/integrate/rest-api/). Everything that you see a Mattermost client doing, your integration can do too with this API.

### I want to build an interactive app

> That's no problem. Our Apps provide a framework with utilities like a KV store, bot account creation, OAuth permission scopes, and rich UX entry points.

Reviewer 2: ```suggestion
That's no problem. Our Apps provide a framework with utilities like a key-value store, bot account creation, OAuth permission scopes, and rich UX entry points.
```

To build a richer bot integration, you can make full use of the [Mattermost REST API](/integrate/rest-api/). Everything that you see a Mattermost client doing, your integration can do too with this API.

> ### I want to develop a cloud-based app 

Reviewer: Cloud based?

Author: I couldn't think of a better way to frame it. How about "I want a more flexible option"?

Reviewer: Isn't the selling point the rich UX possibilities?

> ### I want to develop a cloud-based app 

Author: ```suggestion
### I want to build something with more interactivity
```

Reviewer: Apply this suggestion?
